### PR TITLE
New docs #78

### PR DESCRIPTION
**Pull request by kof**

### CODE REVIEW

**Comment by kof** [comment]
@joshgillies @mpal9000 @wldcordeiro your feedback is very welcome)

**Comment by joshgillies** [comment]
Looks good to me. Sets the scene for further improvement/collaboration.

**Comment by kof** [comment]
If you guys see additional points I need to mention in any category or typos or just bad english, feel free to correct me.

File: docs/js-api.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+## API
+
```
Comment (wldcordeiro) [inline docs/js-api.md]: Add a TOC? I think that'd be helpful for someone just hunting for reference.

**Comment by wldcordeiro** [comment]
Only the one suggestion about adding TOC's to the API sections but otherwise it's a great jumping off point.

**Comment by mpal9000** [comment]
much easier to digest

**Pull request merged**